package.json

```diff
@@ -1,8 +1,8 @@
 {
 	"name": "apache-daffodil-vscode",
-	"displayName": "Apache Daffodil VS Code Extension",
-	"description": "VS Code extension for Apache Daffodil DFDL schema debugging",
-	"version": "1.2.0-SNAPSHOT",
+	"displayName": "Apache Daffodil™ Extension for Visual Studio Code",
+	"description": "Apache Daffodil™ Extension for Visual Studio Code providing DFDL syntax highlighting, DFDL code completion, DFDL schema debugging, and data editor",
+	"version": "1.3.0-SNAPSHOT",
```

Comment:
I would prefer we bump the snapshot version in a separate commit. Also, this branch should have been pushed to a fork and not directly to the apache/daffodil-vscode repo.

Reply:
Okay, I'll roll the SNAPSHOT version out of here and get that in a different PR on my fork.  Noted about doing PRs in forks and not directly on the upstream repo.